app/src/main/java/org/mozilla/tv/firefox/ext/Locale.kt

```diff
@@ -13,6 +14,6 @@ fun Locale.languageAndMaybeCountryMatch(allowedLocales: Array<out Locale>?): Boo
         val languageMatches = allowed.language == this.language
         val countryMatches = allowed.country.isEmpty() ||
                 allowed.country == this.country
-        return languageMatches && countryMatches
+        return@any languageMatches && countryMatches
```

Comment:
Omg. 🤣

Reply:
Yep....

This didn't cause any production bugs btw, but would have if we hadn't caught it here.